test/fixtures/users.yml

```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
```

Comment:
Could we add a fixture for each of the roles that we have discussed? `staff`, `client`, `admin`